packages/ui-components/src/components/Chart/VerticalTick.js

```diff
@@ -17,9 +17,9 @@ export const VerticalTick = ({ x, y, payload }) => {
   return (
     <g transform={`translate(${x - 5},${y + 3})`}>
       <text
-        fontSize="14px"
-        transform="rotate(90)"
-        textAnchor="start"
+        fontSize="13px"
+        transform="rotate(305)"
```

Comment:
I saw @nikhitasinghsk asking whether this was configurable the other day, could be cool in future to expose this through to presentation options